Assets/_Application/Cameras/MoveCameraToCoordinate.cs

```diff
@@ -73,13 +73,15 @@ public void LookAtTarget(Coordinate targetLookAt, double targetDistance)
 
         private void ShiftOriginIfNeeded(Coordinate targetCoordinate)
         {
-            var currentCameraPosition = cameraWorldTransform.Coordinate;
-            var difference = (currentCameraPosition - targetCoordinate).Convert(CoordinateSystem.RD); //use RD since this expresses the difference in meters, so we can use the SqrDistanceBeforeShifting to check if we need to shift.
-            ulong sqDist = (ulong)(difference.easting * difference.easting + difference.northing * difference.northing);
+            Vector3 currentCameraPosition = cameraWorldTransform.Coordinate.ToUnity();
+            Vector3 target = targetCoordinate.ToUnity();
```

Comment:
dit gaat niet goed, je converteert nu eerst naar Unity waardoor je floaring point errors krijgt

Reply:
als je echt wgs84 wil houden kan je ze converteren en easting en northing gebruiken en die van elkaar aftrekken, maar sqDist moet in meters zijn, dus dat zal toch een crs moeten zijn dat in meters is.